### PR TITLE
Awesome books es6

### PR DESCRIPTION
**Pull request by biruk-bereka**
## In this PR I add different features: 
   - Create a collection where users can add a book with author and title.
  - User can remove the collection.
  - Collections are displayed at the top of the page.
  - Used local storage to store the collections.
  - User can navigate to different pages
  - I Use ES6 syntax and code are divided into modules
  - I use Luxon library for date.

### CODE REVIEW

File: modules/navigation.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  contactInfoLink.addEventListener('click', () => {
+    bookListLink.classList.remove('active');
+    bookFormLink.classList.remove('active');
+    contactInfoLink.classList.add('active');
+    contactInfoLink.classList.add('active');
+    bookSection.classList.add('invisible');
+    formSection.classList.add('invisible');
+    contactSection.classList.remove('invisible');
+  });
+};
```
Comment (TedLivist) [inline modules/navigation.js:37]: - [ ] I am unable to visit different pages in your app. Regardless of which navbar menu I select, this is the screen it keeps showing me (see image). Kindly fix that. I am unable to see the form for adding a book, so I cannot check if books are listed or if the contact is displayed.

  image

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @biruk-bereka,

Good job so far!
✔️ Use of ES6 syntax
✔️ Use of luxon for date and time.
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/navigation.js (hunk 1 above)
Comment (biruk-bereka) [reply modules/navigation.js:37]: @TedLivist 

https://user-images.githubusercontent.com/56417904/212906367-55834743-3e0c-415c-8708-ade5932c8a76.mov

Everything is working smoothly on my end. Could you double check that you're viewing the correct branch ('awesome-books-ES6') and using live server? My partner has also confirmed that it's functioning properly on their end as well, as shown in the video. Let me know if you're still experiencing any issues. Thank you.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @biruk-bereka, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ All linters passing
✔️ Professional Readme

 ## Required Changes ♻️
- [ ] As suggested by the previous reviewer, I am not able to navigate to different pages. Also on your deployment via GitHub pages, the same problem persists. Kindly check.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by biruk-bereka** [comment]
@sja-thedude 

https://user-images.githubusercontent.com/56417904/212906367-55834743-3e0c-415c-8708-ade5932c8a76.mov

Everything is working smoothly on my end. Could you double check that you're viewing the correct branch ('awesome-books-ES6') and using live server? My partner has also confirmed that it's functioning properly on their end as well, as shown in the video. Let me know if you're still experiencing any issues. Thank you.

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @biruk-bereka , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**